files/ko/web/api/element/scrollheight/index.md

```diff
@@ -21,18 +21,6 @@ var intElemScrollHeight = element.scrollHeight;
 
 *intElemScrollHeight*은 요소 `scrollHeight`의 픽셀 값을 저장하는 정수형 변수입니다.
 
-## 예제
-
-padding-top
-
-Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-
-Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-
-padding-bottom
-
-**Left** **Top** **Right** **Bottom** _margin-top_ _margin-bottom_ _border-top_ _border-bottom_
-
```

Comment:
Temporarily removed as this part is not relevant

Reply:
- https://github.com/mdn/translated-content/pull/17716/commits/37dd697c6bd5ce59afeac93b93b50008640fa27f